### PR TITLE
Update assetlist.json

### PR DESCRIPTION
**Pull request by JIMMwe**

### CODE REVIEW

File: neutron/assetlist.json [hunk 1]
```diff
@@ -446,6 +446,32 @@
...
+          "denom": "jimmy",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/neutron108x7vp9zv22d6wxrs9as8dshd3pd5vsga463yd/JIMMY",
+      "name": "Jimmy",
+      "display": "Jimmy",
+      "symbol": "JIMMY",
+      "logo_URIs": {
+        "png": "https://jimmy-price.vercel.app/jimmy44.png"
```
Comment (JeremyParish69) [inline neutron/assetlist.json:470]: Please upload the image into this repo

File: neutron/assetlist.json [hunk 2]
```diff
@@ -446,6 +446,32 @@
...
+      },
+      "images": [
+        {
+          "png": "https://jimmy-price.vercel.app/jimmy44.png"
+        }
```
Comment (JeremyParish69) [inline neutron/assetlist.json:475]: Please correct the schema (missing some brackets)

**Comment by JeremyParish69** [comment]
Please use more appropriate PR title

File: neutron/assetlist.json (hunk 1 above)
Comment (JIMMwe) [reply neutron/assetlist.json:471]: sorry, but its uploaded with the url what is needed furthermore?

Comment (JeremyParish69) [reply neutron/assetlist.json:471]: The logo image must be added to the neutron/images/ folder in this repo, and the URL must point to that upload--and not to an external website.

File: neutron/assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply neutron/assetlist.json:476]: The schema is still broken

File: neutron/assetlist.json (hunk 1 above)
Comment (JIMMwe) [reply neutron/assetlist.json:471]: sorry for bothering, but how can i add the image there? i tried creating folder and couldnt drop photo to save it

Comment (JIMMwe) [reply neutron/assetlist.json:471]: also i cant upload file since it needs permission

Comment (JeremyParish69) [reply neutron/assetlist.json:471]: You have full permissions on your own fork. Be sure you're uploading to JIMMwe:patch-1 https://github.com/JIMMwe/chain-registry/tree/patch-1

File: neutron/assetlist.json [hunk 3]
```diff
@@ -446,6 +446,33 @@
...
+      "description": "Jimmy Neutron Finance",
+      "denom_units": [
+        {
+          "denom": "factory/neutron108x7vp9zv22d6wxrs9as8dshd3pd5vsga463yd/JIMMY",
+          "exponent": 0,
+          "aliases": [
+            "ujimmy"
+          ]
+        }
+        },
```
Comment (JeremyParish69) [inline neutron/assetlist.json:460]: this cannot be here

File: neutron/assetlist.json (hunk 1 above)
Comment (JIMMwe) [reply neutron/assetlist.json]: gotcha, done please check again

File: neutron/assetlist.json (hunk 3 above)
Comment (JIMMwe) [reply neutron/assetlist.json]: hey check

**Pull request closed without merging**